lisa/notifiers/text_result.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from dataclasses import dataclass
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+class TextResult(notifier.Notifier):
+    """
+    Creating log notifier to dump text formatted results for easier 
+    view in editing mode.
```

Comment:
The "editing mode" is lack of context here. We need to translate it to be in general. It can explain the original log is completed, but too long to check only the summary.